src/plugins/visualizations/public/vis_types/types.ts

```diff
@@ -41,13 +47,16 @@ export interface VisType<TVisParams = unknown> {
   readonly name: string;
   readonly title: string;
   readonly description?: string;
+  readonly note: string;
   readonly getSupportedTriggers?: () => Array<keyof TriggerContextMapping>;
   readonly isAccessible?: boolean;
   readonly requestHandler?: string | unknown;
   readonly responseHandler?: string | unknown;
   readonly icon?: IconType;
   readonly image?: string;
   readonly stage: 'experimental' | 'beta' | 'production';
+  readonly group: VisGroups;
+  readonly groupTitle: string;
```

Comment:
I wonder if there is a use-case why we'd want to still show the `title` for the visualizations that have a `groupTitle`, i.e. what's the advantage of calling it "Text" in the wizard, but "Markdown" in the visualization listing page?

I wonder if we couldn't instead just change the `title` of those visualizations. The only one where I feel a bit on the fence is Vega, that would then also show up as "Custom visualization" in the visualization list. But maybe if we decide that this is what the user should see in the wizard they should see it everywhere?

Was there some background to making those two names differently?

Reply:
cc @cchaos as far as I understand you were driving for this? Could you maybe elaborate for me here, why we should keep those two separated? Are you worried about a specific visualization or about a specific place where this name is showing?

Reply:
I had the same question here https://github.com/elastic/kibana/pull/79627#issuecomment-714271317
@cchaos suggested to differentiate them. Caroline do you want to elaborate on this 😄

Reply:
Ok I changed the naming from `groupTitle` to `titleInWizard` to be more self explanatory